tests/ZfrOAuth2Test/Server/Grant/AuthorizationGrantTest.php

```diff
@@ -244,7 +255,7 @@ public function testCanCreateTokenResponse($hasRefreshGrant)
 
         $response = $this->grant->createTokenResponse($request, new Client(), $owner);
 
-        $body = json_decode($response->getContent(), true);
+        $body = json_decode($response->getBody(), true);
```

Comment:
Nope sorry getContents still return empty string even with rewind. That's honestly really complicated and not intuitive :'(.